Resources/Prototypes/Nyanotrasen/Entities/Clothing/Belt/belts.yml

```
@@ -47,18 +47,42 @@
+      wakizashi: !type:ContainerSlot
+  - type: ItemSlots
+    slots:
+      katana:
+        name: Katana
+        insertVerbText: sheath-insert-verb
+        ejectVerbText: sheath-eject-verb
+        insertSound: /Audio/SimpleStation14/Items/Handling/sword_sheath.ogg
+        ejectSound: /Audio/SimpleStation14/Items/Handling/sword_unsheath.ogg
+        whitelist:
+          tags:
+          - Katana
```

yamllint

[error] wrong indentation: expected 12 but found 10

(indentation)